sormas-backend/src/main/java/de/symeda/sormas/backend/sample/SampleJurisdictionPredicateValidator.java

```diff
@@ -0,0 +1,81 @@
+/*
+ *  SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ *  Copyright © 2016-2021 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program. If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+package de.symeda.sormas.backend.sample;
+
+import javax.persistence.criteria.CriteriaBuilder;
+import javax.persistence.criteria.Predicate;
+
+import de.symeda.sormas.api.utils.jurisdiction.JurisdictionValidator;
+import de.symeda.sormas.backend.facility.Facility;
+import de.symeda.sormas.backend.user.User;
+
+public class SampleJurisdictionPredicateValidator extends JurisdictionValidator<Predicate> {
```

Comment:
This seems to be only a partial implementation of the jurisdiction check. It misses checking the jurisdiction of associated object.

Reply:
we should talk about this, the checking of the associated object jurisdictions is handled in the service

Reply:
will refactor the predicate validators